Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkEditor.cs

```diff
@@ -5,35 +5,40 @@
 using UnityEngine;
 using VRCAudioLink;
 
-    [CustomEditor(typeof(AudioLink))]
-    public class AudioLinkEditor : Editor
+[CustomEditor(typeof(AudioLink))]
+public class AudioLinkEditor : Editor
+{
+    public override void OnInspectorGUI()
     {
-        public override void OnInspectorGUI()
-        {
-            if (UdonSharpGUI.DrawDefaultUdonSharpBehaviourHeader(target)) return;
-            EditorGUILayout.Space();
-            if (GUILayout.Button(new GUIContent("Link all sound reactive objects to this AudioLink", "Links all UdonBehaviours with 'audioLink' parameter to this object."))) { LinkAll(); }
-            EditorGUILayout.Space();
-            base.OnInspectorGUI();
-        }
+        if (UdonSharpGUI.DrawDefaultUdonSharpBehaviourHeader(target)) return;
+        EditorGUILayout.Space();
+        if (GUILayout.Button(new GUIContent("Link all sound reactive objects to this AudioLink", "Links all UdonBehaviours with 'audioLink' parameter to this object."))) { LinkAll(); }
+        EditorGUILayout.Space();
+        base.OnInspectorGUI();
+    }
 
-        void LinkAll()
+    void LinkAll()
+    {
+        UdonSharpBehaviour[] allUdonSharpBehaviours = FindObjectsOfType<UdonSharpBehaviour>();
+        foreach (var behaviour in allUdonSharpBehaviours)
         {
-            UdonSharpBehaviour[] allUdonSharpBehaviours = FindObjectsOfType<UdonSharpBehaviour>();
-            foreach (var behaviour in allUdonSharpBehaviours)
+            if (behaviour.GetType().GetField("audioLink") != null)
             {
-                if (behaviour.GetType().GetField("audioLink") != null)
+                behaviour.GetType().GetField("audioLink").SetValue(behaviour, target);
+                EditorUtility.SetDirty(behaviour);
+
+                if (PrefabUtility.IsPartOfPrefabInstance(behaviour))
```

Comment:
you still made a change here. is this intentional?

Reply:
![image](https://user-images.githubusercontent.com/11212115/230725514-b3cdf123-a2d5-40f5-ae6e-e6f0c0ee1504.png)

Reply:
oops, it's resolved this time